views/mattermost/server_fact.view.lkml

```diff
@@ -1,5 +1,5 @@
 view: server_fact {
-sql_table_name: mattermost.server_fact ;;
```

Comment:
Are changes in this file intended for this PR?

Reply:
You're right @jasonblais, I thought the changes would create a new PR, I removed these changes from this one.

Reply:
I'll create a new one for these changes, once this PR gets merged.

Reply:
@sri-byte If you create a new git branch and submit your changes for server_fact table there, you can submit a separate PR for this second git branch. That way you don't have to wait until this PR is merged. I can simply approve the one for the server_fact table.

Let me know if you have any questions about using multiple git branches for separate PRs.